AnkiDroid/src/main/java/com/ichi2/libanki/Decks.java

```diff
@@ -138,6 +139,7 @@
     private Collection mCol;
     private HashMap<Long, JSONObject> mDecks;
     private HashMap<Long, JSONObject> mDconf;
+    private HashMap<String, JSONObject> nameMap;
```

Comment:
Should follow convention for member variables

```suggestion
    private HashMap<String, JSONObject> mNameMap;
```